Fabric.Identity.API/Exceptions/ExceptionMessageResources.cs

```diff
@@ -5,6 +5,6 @@ public static class ExceptionMessageResources
         public static string MissingUserClaimMessage = "No User claim present on token";
         public static string MissingIssuerClaimMessage = "No Issuer claim present on token";
         public static string ForbiddenIssuerMessageLog = "The Issuer on the claim ({0}) is not in the Issuer White List";
-        public static string ForbiddenIssuerMessageUser = "Cannot access because the site is not configured for your account. Please contact your administrator for assistance";
+        public static string ForbiddenIssuerMessageUser = "Access denied. Try closing the browser and logging in with a different user, or contact your administrator for assistance.";
```

Comment:
The last sentence is a bit long.  Maybe two sentences:  "Try closing your browser and logging in with a different user.  If you continue to have problems please contact your administrator for assistance"